### ISSUE TITLE
ERROR pool/attestations: Cannot delete seen attestations error=<nil>

### ISSUE THREAD

**Issue by prestonvanloon** [prysmaticlabs/prysm#7236]
# 🐞 Bug Report

### Description

Users are seeing an error message for a nil error after deleting seen attestations.

```
ERROR pool/attestations: Cannot delete seen attestations error=<nil>
```

https://github.com/prysmaticlabs/prysm/blob/572227d25e5c76b6f662e9713aedb7b0c668e6eb/beacon-chain/operations/attestations/prune_expired.go#L38-L40

This is not an actual error, but a mistake in the if statement. The line of code should check `err != nil` but does the opposite.

:rotating_light:  **Users experiencing this error may safely ignore the message for now.** :rotating_light: 

### Has this worked before in a previous version?

Yes.

## 🔬 Minimal Reproduction

Run any beacon chain with alpha.25.

## 🔥 Error

```
ERROR pool/attestations: Cannot delete seen attestations error=<nil>
```

## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
Linux AMD64
  </code>
</pre>

**What version of Prysm are you running? (Which release)**

<pre>
  <code>
v1.0.0-alpha.25
  </code>
</pre>

**Anything else relevant (validator index / public key)?**

**Comment by terencechain** [comment]
@prestonvanloon fixed here https://github.com/prysmaticlabs/prysm/pull/7217

**Comment by prestonvanloon** [comment]
Thanks!

**Comment by prestonvanloon** [comment]
This will be resolved in v1.0.0-alpha.26
